tutorials/url-shortener.mdx

```diff
@@ -99,25 +99,18 @@ framework do I want to use? I think I'm going to go with Rocket (opens new
 window). It's pretty much production ready with a sweet API and I'm reasonably
 proficient with it.
 
-I open up `src/lib.rs` and overwrite it with the shuttle entrypoint code:
-
-```rust
-#[macro_use]
-extern crate rocket;
+Before deploying the app we have to create a new shuttle project which is done as follows
```

Comment:
This is true, but I think this part should be moved down to just before the `cargo shuttle deploy` step. The `init` command should come first, and you can include some snippets showing the `lib.rs` and `Cargo.toml` files it produces. 

Below this the author writes about red syntax highlighting due to missing imports, but this won't be the case now since `cargo shuttle init` also adds a cargo.toml with the correct dependencies. This is the code that `cargo shuttle init --rocket url-shortener` will produce, btw: https://github.com/shuttle-hq/examples/tree/main/rocket/hello-world. The name in `Shuttle.toml` and `Cargo.toml` will be the same as the one passed in to `init`.